IdentityCore/src/controllers/MSIDLocalInteractiveController.m

```diff
@@ -118,9 +118,9 @@ - (void)handleWebMSAuthResponse:(MSIDWebMSAuthResponse *)response completion:(MS
     completionBlock(nil, appInstallError);
 }
 
+#if TARGET_OS_IPHONE
```

Comment:
Why don't we keep it inside `promptBrokerInstallWithResponse` method? Now if we change method declaration we would need to update it in 2 places.

Reply:
I found it weird that a parameter which is used would then in fact be marked as unused, but I can change that

Reply:
We can improve it by removing `__unused` from method declaration and add `(void)response; ` just after `#else` so it will silence the warning.